docs/src/index.md

```diff
@@ -114,55 +142,65 @@ pushforward to find ``\dfrac{∂f}{∂x}``:
 
 !!! note
 
-    There is one `Δarg` per `arg` to the original function.
-    The `Δargs` are similar in type/structure to the corresponding inputs `args` (`Δself` is explained below).
-    The `∂Y` are similar in type/structure to the original function's output `Y`.
-    In particular if that function returned a tuple then `∂Y` will be a tuple of same size.
+    The pullback returns one `∂arg` per `arg` to the original function, plus one `∂self` for the fields of the function itself (explained below).
+
+!!! terminology
+    Sometimes _perturbation_, _seed_, and even _sensitivity_ will be used interchangeably.
+    They are not generally synonymous, and ChainRules shouldn't mix them up.
+    One must be careful when reading literature.
+    At the end of the day, they are all _wiggles_ or _wobbles_.
 
-The pullback is a function:
+
+The pushforward is a part of the `frule` function.
+Considered alone it would look like:
 
 ```julia
-function pullback(ΔY)
+function pushforward(Δself, Δargs...)
     ...
-    return ∂self, ∂args...
+    return ∂y
+end
+```
+But because it is fused into frule we see it as part of:
+```julia
+function frule(::typeof(foo), args..., Δself, Δargs...; kwargs...)
+    ...
+    return y, ∂y
 end
 ```
 
-The input to the pullback is often called the _seed_.
-If the function is `y = f(x)` often the pullback will be written `s̄elf, x̄ = pullback(ȳ)`.
-
-!!! note
 
-    The pullback returns one `∂arg` per `arg` to the original function, plus one for the fields of the function itself (explained below).
+The input to the pushforward is often called the _perturbation_.
+If the function is `y = f(x)` often the pushforward will be written `ẏ = last(frule(f, x, ṡelf, ẋ))`.
+`ẏ` is commonly used to represent the perturbation for `y`.
 
-!!! terminology
-    Sometimes _perturbation_, _seed_, and even _sensitivity_ will be used interchangeably.
-    They are not generally synonymous, and ChainRules shouldn't mix them up.
-    One must be careful when reading literature.
-    At the end of the day, they are all _wiggles_ or _wobbles_.
+!!! note
+    
+    In the `frule`/pushforward,
+    there is one `Δarg` per `arg` to the original function.
+    The `Δargs` are similar in type/structure to the corresponding inputs `args` (`Δself` is explained below).
+    The `∂y` are similar in type/structure to the original function's output `Y`.
+    In particular if that function returned a tuple then `∂y` will be a tuple of the same size.
 
-### Self derivative `Δself`, `∂self`, `s̄elf`, `ṡelf` etc.
+### Self derivative `Δself`, `∂self`, `s̄elf`, `ṡelf` etc.
 
-!!! terminology  `Δself`, `∂self`, `s̄elf`, `ṡelf`
+!!! terminology  "`Δself`, `∂self`, `s̄elf`, `ṡelf`"
     It is the derivatives with respect to the internal fields of the function.
     To the best of our knowledge there is no standard terminology for this.
     Other good names might be `Δinternal`/`∂internal`.
```

Comment:
hmmm, this still doesn't seem to render right... may something like
```julia
!!! terminology "self"
    `Δself`, `∂self`, `s̄elf`, `ṡelf` denote the derivatives with respect to the internal fields of the function.
    To the best of our knowledge there is no standard terminology for this.
    Other good names might be `Δinternal`/`∂internal`.
```

Reply:
I think just dropping the backticks is best